content/source/docs/cloud/integrations/service-now/index.html.md

```diff
@@ -190,3 +194,84 @@ Access Control Roles | Description
 
 For users who only need to order from the Terraform Catalog, we recommend creating another role with read-only permissions for `x_terraform_vcs_repositories` to view the available repositories for ordering infrastructure.
 
+## Example Customizations
+
+### Creating a Catalog Item with Pinned Variables
+
+This example use case is used to demo a potential solution to creating a Terraform Catalog Item for resources that limits user input to certain variables.
+
+##### Create Service Catalog Item
+
+1. Enter the ServiceNow Studio
+1. Click "Create Application File"
+    - Select "Catalog Item" under "Service Catalog" > Click "Create"
+1. Name the new Catalog Item. (Our example will be `Example With Pinned Variables`)
+1. Select Catalogs: "Terraform Catalog" > Select Categories: "Terraform Resources"
+1. Add any other descriptions you may want.
+1. Click "Submit"
+
+##### Create Variable Set
+
+1. Click "Create Application File"
+1. Select "Variable Set" > "Service Catalog" > Click "Create"
+1. Name your variable set (Our example will be `Example Pinned Variables`)
+    - This will default "Internal Name" to `example_pinned_variables`
+1. Click "Submit"
+1. Under the "Variables" tab click "New"
+    1. Create your variable:
+    1. In this example we will create a field called `tf_var_pet_name_length` that will be for a Terraform variable that determines the number of words to use for the pet server name.
+        - Question: `Pet Name Length`
+        - Name: `tf_var_pet_name_length`
+        - `tf_var_` tells the Terraform ServiceNow SDK that this is a Terraform Variable.
+    1. Repeat variable creation as necessary for your use case.
+    1. Click "Submit"
+
+##### Add Variable Set to Catalog Item
+
+1. Go back to your Catalog Item
+1. Select "Variable Sets Tab" > Click "New"
+1. Search for the Variable Set that was created (`Example Pinned Variables`)
+1. Click "Submit"
+
+##### Create Custom Workflow
+
+We will use the "Example With Pinned Variables" example workflow for our custom workflow.
+
+In the ServiceNow Studio:
+
+1. Open the "Example With Pinned Variables", Workflow > Workflow > "Example With Pinned Variables"
+1. Double click on the "Run Script" item
+1. Copy the script code from the editor
+1. Close the "Run Script" and "Workflow" window
+1. Create a new custom workflow: Create Application File > Workflow
+    - Name: `Example Pinned Variables` (replace with your custom workflow name)
+    - Table: `Requested Item [sc_req_item]` and Click "Submit"
+1. Add a "Run Script" item. Core Tab > Utilities > Run Script
+1. Name: `Example Workflow Pinned Variables`
+1. Script:
+    - Paste in the code copied from "Provision Resources with Variables"
```

Comment:
This is the first time this is mentioned in this process. Did you miss adding something above, or mis-copy a name?

Reply:
It was mentioned a few steps above. I made an edit to hopefully clarify it better.